source/reference/mongosync.txt

```diff
@@ -34,39 +35,44 @@ Global Options
 .. option:: --cluster0 <URI>
 
    Sets the :ref:`connection URI <mongodb-uri>` for the first cluster.
-   The first cluster can serve as either the source cluster or the destination 
-   cluster in the sync process.
+   The first cluster can serve as either the source cluster or the
```

Comment:
Maybe too many clusters in this sentence. Consider something like:
You can specify the first cluster as either the source or destination cluster for the sync, using the call to the ...